pkg/builder/controller.go

```diff
@@ -304,13 +308,25 @@ func (blder *Builder) doController(r reconcile.Reconciler) error {
 		ctrlOptions.CacheSyncTimeout = *globalOpts.CacheSyncTimeout
 	}
 
+	controllerName := blder.getControllerName(gvk)
+
 	// Setup the logger.
-	if ctrlOptions.Log.GetSink() == nil {
-		ctrlOptions.Log = blder.mgr.GetLogger()
+	if ctrlOptions.LogConstructor == nil {
+		log := blder.mgr.GetLogger()
+		log = log.WithValues("reconcilerGroup", gvk.Group, "reconcilerKind", gvk.Kind)
```

Comment:
Not sure if we need reconcilerGroup + reconcilerKind, but I'm fine with keeping them